components/script/dom/element.rs

```diff
@@ -179,6 +180,14 @@ pub enum CustomElementCreationMode {
     Asynchronous,
 }
 
+#[derive(Clone, Copy, PartialEq, Eq, HeapSizeOf, JSTraceable)]
+pub enum CustomElementState {
```

Comment:
Add a link to https://dom.spec.whatwg.org/#concept-element-custom-element-state .